gym_cap/envs/cap_env.py

```diff
@@ -385,7 +385,7 @@ def seed(self, seed=None):
         """
         self.np_random, seed = seeding.np_random(seed)
         return [seed]
-
+      
```

Comment:
Try to remove commits that is not part of the change

Reply:
removed commit